### DISCUSSION TITLE
Cambiare la cartella di configurazione

### DISCUSSION THREAD

**Discussion by array81** [indomus/forum#1748] [HUB: Home Assistant]
Mi sono avvicinato da qualche settimana a Home Assistant. L'ho installato su un RaspBerry 3B+, in particolar modo ho installato Home Assistant Core su Docker che gira su Raspberry OS (64bit).
Mi sono accorto di aver commesso un errore che non pregiudica a quanto pare il funzionamento di Home Assistant ma che vorrei sistemare se possibile. In sostanza seguendo la guida per l'installazione non ho sostituito il percorso della cartella dove Home Assistant salva le configurazioni e i vari componenti custom.
Questa attualmente è settata su /pi/homeassistant mentre io vorrei fosse all'interno della mia cartella utente /mioutente/homeassistant, ho infatti rinominato il nome utente all'atto dell'installazione di Raspberry OS. Mi potete spiegare come fare senza dover reinstallare il tutto?

Grazie.

**Comment by Scialla** [comment]
Ti basta copiare il contenuto di pi/homeassistant in /mioutente/homeassistant e cambiare la mappatura delle config nella creazione del container; ovviamente lo stoppi e lo ricrei poi con la nuova mappatura
se usi docker-compose è un attimo, se crei il container docker a mano, pure :)
